plugins/pitchfork.py

```diff
@@ -0,0 +1,39 @@
+import re
+import string
+import textwrap
+
+from errbot import BotPlugin, botcmd
+
+
+class Pitchfork(BotPlugin):
+    """
+    To pitchfork users down to ...
+    """
+
+    @botcmd
+    def pitchfork(self, msg, arg):
+        """
+        To pitchfork user down to ...
+        """
+        match = re.match(r'@?([\w-]+)(?:\s+(?:down\s+)?to\s+(.+))?$',
+                         arg)
+        if match:
+            user = match.group(1)
+            place = match.group(2) if match.group(2) else 'offtopic'
+            return textwrap.dedent((
+                string.Template("""
+                    @$user, you are being pitchforked down to $place
```

Comment:
any reason to use `string.Template` over https://pyformat.info?

Reply:
yes, because the string contains `{` and `}` and there's no way to escape those(can't believe it)

Reply:
um you can put `{{` `}}` to escape them, but this is fine, I guess